boardStupid.php

```diff
@@ -19,14 +14,18 @@
         <title>boardStupid</title>
     </head>
     <body>
-    <form method="post">
+    <form action="bs_update_db.php" method="post">
         <label>Game:<br /> <input type="text" name="name"/></label><br />
```

Comment:
So Ash told me off for using break tags. He suggested putting each section you want on separate lines into divs because they are block elements (and it also means you can style it more easily later)